membrane_curvature/base.py

```diff
@@ -0,0 +1,126 @@
+"""
+MembraneCurvature
+=======================================
+
+:Author: Estefania Barreto-Ojeda
+:Year: 2021
+:Copyright: GNU Public License v3
+
+MembraneCurvature calculates the mean and Gaussian curvature of
+surfaces derived from a selection of reference.
+"""
+
+import numpy as np
+import warnings
+from membrane_curvature.surface import get_z_surface
+from membrane_curvature.curvature import mean_curvature, gaussian_curvature
```

Comment:
Use relative imports
```suggestion
from .surface import get_z_surface
from .curvature import mean_curvature, gaussian_curvature
```
to be explicit about the relationship between the modules. This makes clear that they are all in the same package.